drivers/builtin/src/rsa.c

```diff
@@ -2763,6 +2764,13 @@ int mbedtls_rsa_self_test(int verbose)
     MBEDTLS_MPI_CHK(mbedtls_mpi_read_string(&K, 16, RSA_E));
     MBEDTLS_MPI_CHK(mbedtls_rsa_import(&rsa, NULL, NULL, NULL, NULL, &K));
 
+#if !defined(MBEDTLS_RSA_NO_CRT)
+    MBEDTLS_MPI_CHK(mbedtls_rsa_deduce_crt(&rsa.P, &rsa.Q, &rsa.D, &DP, &DQ, &QP));
```

Comment:
Minor: why do we need intermediate variables here? Why not directly use `&rsa.DP` etc as the output parameters in this call?